nixos/postgresql: take extraPlugins packageset from package option (#257503)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2405.section.md

- Cinnamon has been updated to 6.0. Please beware that the [Wayland session](https://blog.linuxmint.com/?p=4591) is still experimental in this release.

> - `services.postgresql.extraPlugins` changed it's type from a list of packages to a function that returns the former.

Reviewer: ```suggestion
- `services.postgresql.extraPlugins` changed its type from a list of packages to a function that returns the former.
```

> - `services.postgresql.extraPlugins` changed its type from a list of packages to a function that returns the former.
>   For example a config line like ``services.postgresql.extraPlugins = with pkgs.postgresql_11.pkgs; [ postgis ];`` would need to be changed to ``services.postgresql.extraPlugins = ps: with ps; [ postgis ];``;
> 

Reviewer: ```suggestion
- `services.postgresql.extraPlugins` changed its type from just a list of packages to also a function that returns such a list.
  For example a config line like ``services.postgresql.extraPlugins = with pkgs.postgresql_11.pkgs; [ postgis ];`` is recommended to be changed to ``services.postgresql.extraPlugins = ps: with ps; [ postgis ];``;

```
